service/policy/resourcemapping/resource_mapping.go

```diff
@@ -35,6 +35,82 @@ func NewRegistration() serviceregistry.Registration {
 	}
 }
 
+/*
+	Resource Mapping Groups
+*/
+
+func (s ResourceMappingService) ListResourceMappingGroups(ctx context.Context, req *resourcemapping.ListResourceMappingGroupsRequest) (*resourcemapping.ListResourceMappingGroupsResponse, error) {
```

Comment:
Can you please open an issue for Audit of the the Create/Update/Delete functionality for Resource Mapping Groups to align with other policy objects? Right now I don't think we're auditing any read events but having that trail for writes is helpful to admins.

Reply:
We should also make sure we're aligned with utilization of a service logger pointer receiver throughout to associate our logs with the service sending them: https://github.com/opentdf/platform/blob/5b9e2cfe3607a4412550c6f5f3f3f699d918734a/service/policy/resourcemapping/resource_mapping.go#L19

Reply:
https://github.com/opentdf/platform/issues/1370